server/index.js

```diff
@@ -4,16 +4,27 @@ const cors = require('cors');
 const app = express();
 const apiPort = 8080;
 
-const db = require('./db');
+//connected to mongoose through db/index.js
+const { getMongoDB, uri } = require('./db');
```

Comment:
This could actually go into the if statement as well

Reply:
ype